app/workers/notify_end_rent_worker.rb

```diff
@@ -0,0 +1,14 @@
+require 'sidekiq-scheduler'
+
+class NotifyEndRentWorker
+  include Sidekiq::Worker
+
+  def perform
+    logger.info 'Verifying rents...'
+    Rent.where(end_date: Time.zone.now).pluck(:id).each do |rent_id|
```

Comment:
This only works if this scheduler runs every second, it's better if you add a "notified" field to rent and check when end_date is older than Time.zone.now and it's not notified.

Reply:
right! done!